### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by bramceulemans**
Add .idea/ folder to .gitignore

### CODE REVIEW

**Comment by Raqbit** [comment]
LGTM!

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = crlf
```
Comment (Raqbit) [inline .editorconfig]:
```suggestion
end_of_line = lf
```

Comment (bramceulemans) [reply .editorconfig]: Fixed.

**Comment by Raqbit** [comment]
Thanks!

**Pull request merged**